src/components/Player/playerPages.jsx

```diff
@@ -90,7 +90,8 @@ const playerPages = strings => [{
   content: (playerId, routeParams, location) => (<ActivityPage playerId={playerId} routeParams={routeParams} location={location} />),
 }];
 
-export default (playerId, strings) => playerPages(strings).map(page => ({
+export default (playerId, strings, isPlayerProfilePrivate) => playerPages(strings).map(page => ({
   ...page,
   route: `/players/${playerId}/${page.key}`,
+  disabled: isPlayerProfilePrivate,
```

Comment:
where is this getting used?

Reply:
![image](https://github.com/odota/web/assets/65485512/4f5714e9-ffb8-4d00-b377-24d58228279d)

The existing "Tab" component accepts a "disabled" prop. The idea here is to disabled all the tabs if the profile is private.

So in "playerPages.jsx", where the pages are being mapped to the tabs, I added in an extra "disabled" property that will eventually be passed to the "Tab" component